config/routes.rb

```diff
@@ -13,6 +13,7 @@
 
       resources :topics, only: :index
       resources :spots, only: %i[create index destroy]
+      resources :matches, only: :index
```

Comment:
I think the route should be ```/user/matches```
In order to do this move this route inside the devise user scope.